draft-ietf-tls-tls13.md

```diff
@@ -898,8 +897,6 @@ items (some of which are not required by all ciphers, and are thus empty):
 
```

Comment:
"following four items" should be "following two items" now

Alternatively, just change line 895-900 to:
"The record layer will use the security parameters to generate the client and server write keys."
(with updated/removed parenthetical)